arkouda_workflows/deploy-arkouda-on-kubernetes-cronworkflow.yaml

```diff
@@ -0,0 +1,412 @@
+  
+      - name: create-locale-ssh-service
+        resource:
+          action: create
+          manifest: |
+            apiVersion: v1
+            kind: Service
+            metadata:
+              name: {{ workflow.parameters.arkouda-instance-name }}-locale-ssh
+            spec:
+              type: ClusterIP
+              ports:
+                - port: 22
+              selector:
+                app: {{ workflow.parameters.arkouda-instance-name }}-locale
+
+      - name: create-pod-role
+        resource: 
+          action: create
+          manifest: |
+            apiVersion: rbac.authorization.k8s.io/v1
+            kind: Role
+            metadata:
+              name: "{{ workflow.parameters.arkouda-instance-name }}-pod"
+            rules:
+              - apiGroups: [""]
+                resources: ["pods","deployments"]
+                verbs: ["get", "watch", "list"]
+ 
+      - name: create-pod-role-binding
+        resource:
+          action: create
+          manifest: |
+            apiVersion: rbac.authorization.k8s.io/v1
+            kind: RoleBinding
+            metadata:
+              name: "{{ workflow.parameters.arkouda-instance-name }}-pod-binding"
+            subjects:
+            - kind: User
+              name: "{{ workflow.parameters.arkouda-user }}"
+              apiGroup: rbac.authorization.k8s.io
+            roleRef:
+              kind: Role
+              name: "{{ workflow.parameters.arkouda-instance-name }}-pod"
+              apiGroup: rbac.authorization.k8s.io
+
+      - name: create-service-role
+        resource:
+          action: create
+          manifest: |
+            apiVersion: rbac.authorization.k8s.io/v1
+            kind: Role
+            metadata:
+              name: {{ workflow.parameters.arkouda-instance-name }}-service
+            rules:
+              - apiGroups: [""]
+                resources: ["services","endpoints"]
+                verbs: ["get","watch","list","create","delete","update"]
+
+      - name: create-service-role-binding
+        resource:
+          action: create 
+          manifest: |
+            kind: RoleBinding
+            apiVersion: rbac.authorization.k8s.io/v1
+            metadata:
+              name: {{ workflow.parameters.arkouda-instance-name }}-service-binding
+            subjects:
+            - kind: User
+              name: {{ workflow.parameters.arkouda-user }}
+              apiGroup: rbac.authorization.k8s.io
+            roleRef:
+              kind: Role
+              name: {{ workflow.parameters.arkouda-instance-name }}-service
+              apiGroup: rbac.authorization.k8s.io
+
+      - name: create-server-launch-script-config-map
+        resource:
+          action: create
+          manifest: |
+            kind: ConfigMap
+            apiVersion: v1
+            metadata:
+              name: arkouda-server-launch-script
+              labels:
+                name: arkouda-server-launch-script
```

Comment:
nevermind it seems the `create-server-launch-script-config-map` also doesn't rely on `{{ workflow.parameters.arkouda-instance-name }}` so it seems like it should all be good